WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -95,4 +122,31 @@
   });
 
   $('#display-<%= @display %>').button('toggle');
+
+  // From http://stackoverflow.com/questions/901115/how-can-i-get-query-string-values-in-javascript/901144#901144
+  function getParameterByName(name, url) {
+    if (!url) url = window.location.href;
+    name = name.replace(/[\[\]]/g, "\\$&");
+    var regex = new RegExp("[?&]" + name + "(=([^&#]*)|&|#|$)"),
+        results = regex.exec(url);
+    if (!results) return null;
+    if (!results[2]) return '';
+    return decodeURIComponent(results[2].replace(/\+/g, " "));
+  }
+
+  function display_selectors() {
+    $(".admin-selectors, .map-selectors").hide();
+    $display_class = "." + getParameterByName('display') + '-selectors';
+    $($display_class).toggle();
+    //$($display_class).css('display','inline-block');
+  }
+
+  $(function(){
+    display_selectors();
+  });
+
+  // From http://stackoverflow.com/a/14038050/5656749
+  $(document).ajaxStop(function(){
+    display_selectors();
+  });
```

Comment:
@pedrosino: sidenote totally unrelated to the PR; you can put backquotes around `@display` to avoid cc-ing the github user named "display" :p